csrankings.csv

```diff
@@ -10908,7 +10908,7 @@ Sean Warnick,Brigham Young University,https://cs.byu.edu/faculty/sean,k7WBrxoAAA
 Sebastian Boring,University of Copenhagen,http://www.diku.dk/english/staff/?pure=en/persons/431157,Y_QMLKAAAAAJ
 Sebastian Erdweg,TU Delft,http://www.informatik.uni-marburg.de/~seba,2jCQrk4AAAAJ
 Sebastian Fischmeister,University of Waterloo,https://uwaterloo.ca/embedded-software-group/people-profiles/sebastian-fischmeister,NOSCHOLARPAGE
-Sebastian G. Elbaum,University of Nebraska,http://csce.unl.edu/~elbaum,swPW5FYAAAAJ
+Sebastian G. Elbaum,University of Virginia,http://csce.unl.edu/~elbaum,swPW5FYAAAAJ

```

Comment:
There's no information on here pointing to a change in affiliation for Sebastian - can you point to something official (or from Sebastian)? Thanks.

Reply:
Would this suffice? 
https://publicsearch.people.virginia.edu/person/se4ja
Thanks.

Reply:
Perfect, thanks.